SharedCode/src/commonMain/kotlin/ApplicationContract.kt

```diff
@@ -4,10 +4,12 @@ import kotlinx.coroutines.CoroutineScope
 
 interface ApplicationContract {
     interface View {
-        fun setLabel(text: String)
+        fun setStationNames(stationNames: List<String>)
+        fun setArrivalTimeLabel(text: String)
```

Comment:
It would be good to update MainActivity.kt to have these functions so it won't error, then add a comment saying e.g. `TODO: Implement these functions` to make it clear these need to be implemented at some point